web/src/lib/stores/assets.store.ts

```diff
@@ -153,15 +153,19 @@ export class AssetStore {
     this.emit(true);
   }, 10_000);
 
-  async init(viewport: Viewport, ascendingOrder?: boolean) {
+  async init(viewport: Viewport) {
     this.initialized = false;
     this.timelineHeight = 0;
     this.buckets = [];
     this.assets = [];
     this.assetToBucket = {};
     this.albumAssets = new Set();
 
-    const buckets = await getTimeBuckets({ ...this.options, key: getKey(), ascendingOrder });
+    const buckets = await getTimeBuckets({
+      ...this.options,
+      key: getKey(),
+      ascendingOrder: this.options.ascendingOrder,
```

Comment:
this is already included in line 165. What are you adding it again here?

Reply:
oops I forgot to remove it